src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -87,7 +87,7 @@ export default {
 		/**
 		 * Unique id attribute of the input to label
 		 */
-		id: {
+		inputId: {
```

Comment:
This `id` is only used in `aria-labelledby` in `NcCheckboxRadioSwitch`. And for this component it makes no sense what is `inputId`: it is not directly connected to any input. It sets an id based on prop so that `NcCheckboxRadioSwitch` can using the same pattern build the same id and use it `aria-labelledby`.

Can we instead keep it `id` (because it actually sets the id of the root element) and just pass `${id}-label` from `NcCheckboxRadioSwitch`.

Then there also will be only one component that build string `${id}-label`, not two.

Reply:
Makes sense, adjusted the code!